History.md

```diff
@@ -1,3 +1,7 @@
+0.16.0 / 2022-11-17
+===================
+- Allow multi-value headers in request and response
+
```

Comment:
Let's also mention the breaking change to `show`.

Reply:
Ah yes, thanks for the reminder!

Reply:
I'll have to rebase and force push so that the tag points to the latest changes.